test/helper.rb

```diff
@@ -19,6 +26,12 @@ def samples_path
 end
 
 def sample_blob(name)
-  name = File.join(samples_path, name) unless name =~ /^\//
-  Linguist::FileBlob.new(name, samples_path)
+  filepath = (name =~ /^\//)? name : File.join(samples_path, name)
+  Linguist::FileBlob.new(filepath, samples_path)
+end
```

Comment:
Is this still used anywhere? It took me a while to parse the difference between `fixture_blob`, `fixture_blob_memory`, `sample_blob`, and `sample_blob_memory`. I get it now, but I'm not sure if there's a way to simplify that.